src/components/cards/card.tsx

```diff
@@ -0,0 +1,52 @@
+import classNames from "classnames";
+import { TransactionI } from "../../data/transactions";
+import { getCurrencyFormat } from "../../helpers/currency";
+import { CurrencyI } from "../../data/authentication";
+import { DebtI } from "../../data/debts";
+
+interface TransactionProps {
+  content: TransactionI | DebtI;
+  currency?: CurrencyI;
+}
+
+export const Card = ({ content, currency }: TransactionProps) => {
```

Comment:
This is a Card component, so it ok that this component is called like that, the other ones does not make sens that have the name card